README.rst

```diff
@@ -17,3 +17,63 @@ Xero accounting data, and also to allow you to analyse it with traditional tools
 Excel along with novel approaches you may wish to take using LLMs.
 
 __ https://www.xero.com/
+
+Documentation
+-------------
+
+Full documentation is available at https://xerotrust.readthedocs.io/en/latest/
```

Comment:
Not sure we want this, just want to get people over to the main docs as soon as possible rather than duplicating and bit-rotting here...